src/core/public/workspace/index.ts

```diff
@@ -0,0 +1,6 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export { WorkspacesStart, WorkspacesService, WorkspacesSetup } from './workspaces_service';
```

Comment:
shall we add a `README.md` to talk about some internals and usage of workspace service like https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/core/public/chrome

Reply:
Good point, we have a separate PR that adds the documentation of workspace feature design and some implementation details https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5282, I can add the one more section there about the `WorkspacesService`